styles.css

```diff
@@ -0,0 +1,71 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+}
+
+.style {
+    font-family: Courier;
+}
+
+header {
+    background-image: url(code_skills.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    height: 1080px;
+}
+
+.container {
+    margin: 0 auto;
+    width: 1200px;
+    overflow: auto;
+}
+
+ul {
+    overflow: hidden;
+}
+
+li {
+    float: right;
+    list-style: none;
+    margin-right: 10px;
+}
+
+.nav-buttons a:link, .nav-buttons a:visited {
+    color: white;
+    text-decoration: none;
+    text-align: center;
+    padding: 10px 10px;
+}
+
+.nav-buttons a:hover {
+    color: black;
+    text-decoration: none;
+    text-align: center;
+    padding: 10px 0px 10px 0px;
+    background-color: white;
+    border-style: solid;
+    border-color: white;
+}
+
+section {
```

Comment:
Section and header have the same height - could they share a class?

Reply:
I don't think they can at this stage but will try to combine them once I've added more content to see if it fits